buffer/src/main/java/io/netty/buffer/ByteBufUtil.java

```diff
@@ -756,7 +762,7 @@ static ByteBuf encodeString0(ByteBufAllocator alloc, boolean enforceHeap, CharBu
         }
     }
 
-    static String decodeString(ByteBuf src, int readerIndex, int len, Charset charset) {
+    static String decodeStringOld(ByteBuf src, int readerIndex, int len, Charset charset) {
```

Comment:
I will remove this one before merging. Its just there as a left over to be able to run the benchmark and check old vs new implementation

Reply:
Fixed... it was in the first benchmark I took... Messed up the second. Re-running now